components/homme/cmake/HommeMacros.cmake

```diff
@@ -122,10 +122,10 @@ macro(createTestExec execName execType macroNP macroNC
 
   target_link_libraries(${execName} csm_share)
 
-  IF (CXXLIB_SUPPORTED_CACHE)
-    MESSAGE(STATUS "   Linking Fortran with -cxxlib")
-    TARGET_LINK_LIBRARIES(${execName} -cxxlib)
-  ENDIF ()
+  # IF (CXXLIB_SUPPORTED_CACHE)
```

Comment:
Skyler will figure out why this is an issue on weaver, if not anything else, there will be an if statement 'if not WEAVER'...